React/Views/RCTScrollView.m

```diff
@@ -725,6 +725,28 @@ - (void)scrollViewDidEndZooming:(UIScrollView *)scrollView withView:(UIView *)vi
   RCT_FORWARD_SCROLL_EVENT(scrollViewDidEndZooming:scrollView withView:view atScale:scale);
 }
 
+- (void)scrollViewDidEndDecelerating:(UIScrollView *)scrollView
+{
+  // Fire a final scroll event
+  _allowNextScrollNoMatterWhat = YES;
+  [self scrollViewDidScroll:scrollView];
```

Comment:
I agree that we should send a final onScroll event. It's what happens natively, and the fact we don't do this currently is effectively a bug that has been introduced by the throttling mechanism.